### PR TITLE
Cleanup NuGet package workflows

### PR DESCRIPTION
**Pull request by dstenroejl**
## Description

We want to align NuGet package workflows in all domains to follow the same model/principles as in geh-core.

We have also updated all actions/workflows from .github to use the new v8 tag.

## References

https://app.zenhub.com/workspaces/mighty-ducks---the-outlaws-6193fe815d79fc0011e741b1/issues/energinet-datahub/the-outlaws/331

### CODE REVIEW

File: .github/workflows/clients-publish.yml [hunk 1]
```diff
@@ -11,48 +11,105 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-name: Publish Clients to Nuget.org
+
+name: Publish Clients and Clients.Registration to Nuget.org
```
Comment (HenrikSommer) [inline .github/workflows/clients-publish.yml:15]: Now publishes all (*two so far...) packages, so maybe change the filename to reflect the change.

Comment (dstenroejl) [reply .github/workflows/clients-publish.yml:15]: Agree, but I cannot come up with a great name that embraces the libraries. Any suggestion?

File: codecov.yml [hunk 2]
```diff
@@ -1,3 +1,14 @@
+flag_management:
+  default_rules: # the rules that will be followed for any flag added, generally
+    carryforward: true
+  individual_flags: # exceptions to the default rules above, stated flag by flag
+    - name: business
+      paths:
+      - source/GreenEnergyHub.Charges
```
Comment (HenrikSommer) [inline codecov.yml:7]: Can paths be excluded as well? it doesn't make much sense to run coverage on test-projects and other projects only used by the test-projects.

File: source/Energinet.Charges.Libraries/source/Energinet.DataHub.Charges.Clients.Registration/Energinet.DataHub.Charges.Clients.Registration.csproj [hunk 3]
```diff
@@ -28,7 +28,7 @@ limitations under the License.
   <!-- Configuration for NuGet package -->
   <PropertyGroup>
     <PackageId>Energinet.DataHub.Charges.Clients.Registration</PackageId>
-    <PackageVersion>3.0.7$(VersionSuffix)</PackageVersion>
+    <PackageVersion>3.0.8$(VersionSuffix)</PackageVersion>
```
Comment (HenrikSommer) [inline source/Energinet.Charges.Libraries/source/Energinet.DataHub.Charges.Clients.Registration/Energinet.DataHub.Charges.Clients.Registration.csproj:31]: A release note must be added to _.\documents\release-notes\release-notes.md_ as well

File: source/Energinet.Charges.Libraries/source/Energinet.DataHub.Charges.Clients/Energinet.DataHub.Charges.Clients.csproj [hunk 4]
```diff
@@ -28,7 +28,7 @@ limitations under the License.
   <!-- Configuration for NuGet package -->
   <PropertyGroup>
     <PackageId>Energinet.DataHub.Charges.Clients</PackageId>
-    <PackageVersion>3.0.7$(VersionSuffix)</PackageVersion>
+    <PackageVersion>3.0.8$(VersionSuffix)</PackageVersion>
```
Comment (HenrikSommer) [inline source/Energinet.Charges.Libraries/source/Energinet.DataHub.Charges.Clients/Energinet.DataHub.Charges.Clients.csproj:31]: A release note must be added to _.\documents\release-notes\release-notes.md_ as well

File: .github/workflows/clients-publish.yml (hunk 1 above)
Comment (HenrikSommer) [reply .github/workflows/clients-publish.yml:15]: Maybe _charge-libraries-publish.yml_?

Comment (prtandrup) [reply .github/workflows/clients-publish.yml:15]: Or Charge in plural? :) charges-libraries-publish.yml?

Comment (dstenroejl) [reply .github/workflows/clients-publish.yml]: Changed to charges-libraries-publish.yml

File: source/Energinet.Charges.Libraries/source/Energinet.DataHub.Charges.Clients.Registration/Energinet.DataHub.Charges.Clients.Registration.csproj (hunk 3 above)
Comment (dstenroejl) [reply source/Energinet.Charges.Libraries/source/Energinet.DataHub.Charges.Clients.Registration/Energinet.DataHub.Charges.Clients.Registration.csproj:31]: I will add a note

File: source/Energinet.Charges.Libraries/source/Energinet.DataHub.Charges.Clients/Energinet.DataHub.Charges.Clients.csproj (hunk 4 above)
Comment (dstenroejl) [reply source/Energinet.Charges.Libraries/source/Energinet.DataHub.Charges.Clients/Energinet.DataHub.Charges.Clients.csproj:31]: I will add a note

File: codecov.yml (hunk 2 above)
Comment (dstenroejl) [reply codecov.yml:7]: According to the documentation we cannot do it per flag, but I have tested the "ignore" section and it seems to work for flags as well.

**Review by HenrikSommer — APPROVED** [review]

**Pull request merged**